drivers/src/memory_layout.rs

```diff
@@ -63,6 +67,7 @@ pub const ROM_DATA_SIZE: u32 = 996;
 pub const MAN1_SIZE: u32 = 6 * 1024;
 pub const MAN2_SIZE: u32 = 6 * 1024;
 pub const FHT_SIZE: u32 = 2 * 1024;
+pub const IDEVID_MLDSA_PUB_KEY_MAX_SIZE: u32 = 3 * 1024;
```

Comment:
Are we rounding to 3K for alignment? I think the key size is always exactly 2592 bytes, which is 32-byte aligned at least.

Reply:
See comment above. I think the 1K alignment was just to make maintaining this file easier. But I think it's prefferable to just use the PersistentData struct. And then we can just keep things word-aligned and call it good.

But we haven't merged that change in `main` yet because we're running into a couple test failures. So I'm ok if we want to just do this for now and change it to the new model in a future PR.

Reply:
@swenson 3K alignment is for easier maintenance. We'll switch to the new model with Jordan's change.